### ISSUE TITLE
deleting images

### ISSUE THREAD

**Issue by CodeMyst** [hauxir/imgpush#17]
could a feature be added for deleting existing images by their filename?

**Comment by hauxir** [comment]
I guess it could be added but there would need to be a way to authenticate users for that to happen

**Comment by lucacastelnuovo** [comment]
Could a system with an app_key work? Which could also be required for uploading.

**Comment by wassimoo** [comment]
I think authentication should be managed by the API or whatever service using imgpush.

**Comment by lucacastelnuovo** [comment]
Yes, you are right.
Just an `DELETE https://example.com/UUID.png` route would be awesome.

Maybe then add a feature flag in ENV to disable the deletion route, for users who haven't implemented authentication.

**Comment by pdelaby** [comment]
Good idea ! I'm currently working on it, and will test if for some time. 
I'm trying to do as many checks as possible before proceeding the actual delete, to avoid  `DELETE https://my.host/.` removing an entire folder and things like that : 
* checking the param agains basic regex
* checking if the path exist and is a file 

For the authentication, I'm experimenting with Traefik (auth + block call from outside the stack) and will share the working configuration.

**Comment by rafael1209** [comment]
I’ve seen previous discussions about adding a DELETE /:filename route to remove uploaded images.
I wanted to ask if there is currently any supported way to delete images (via API or configuration), or if this still needs to be implemented manually (e.g. via file system cleanup or a custom wrapper service).

**Comment by hauxir** [comment]
implemented in https://github.com/hauxir/imgpush/pull/42
